tests/src/Kernel/ResolverBuilderTest.php

```diff
@@ -21,6 +18,7 @@ class ResolverBuilderTest extends GraphQLTestBase {
   public static $modules = [
     'graphql',
     'typed_data',
```

Comment:
not a problem of this patch, but we also don't need to depend on graphql and typed_data. That is already done by the base class.